rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingSpec.scala

```diff
@@ -171,7 +172,7 @@ class CostAccountingSpec extends FlatSpec with Matchers with PropertyChecks with
             remainder /= 4
             joins += f"_ ${arrow} @${ch}"
           }
-          val joinStr = joins.mkString("; ")
+          val joinStr = joins.toSet.mkString("; ")
```

Comment:
This function is generator of Rholang terms. Each index number corresponds to different terms so conversion to Set means to make some of the indexes equal.

What is the reason for this change?

Reply:
https://rchain.atlassian.net/browse/RCHAIN-3917?
Rholang doesn't support it.
Same joins means `consuming two messages from the same channel`

Reply:
> Every number gets decoded into a unique term

If we change internal behavior of this function, every number will not produce a unique term.
So to get terms without joins on the same channel these numbers must be skipped.

Reply:
Could you elaborate?
What specific do you want?

Reply:
If you want to skip some Rholang terms it's better to find the range of numbers which generates these terms and not used them in tests.

Reply:
Is it realistic to do that? Is it necessary? 
What is the shortcoming of the current solution? 
And how to get those numbers? Iterate all numbers and hardcode here?